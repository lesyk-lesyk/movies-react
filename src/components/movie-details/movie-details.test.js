import React from 'react';
import { MovieDetails } from './index';
import renderer from 'react-test-renderer';

describe('MovieDetails', () => {
  it('renders', () => {
    const movie = {
      "title": "title",
      "tagline": "tagline",
      "vote_average": 8,
      "release_date": "2007",
      "poster_path": "http://poster_path",
      "overview": "overview",
      "genres": [
        "genres", "genres"
      ],
      "runtime": 122
    };

    const tree = renderer
      .create(<MovieDetails movie={movie} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
