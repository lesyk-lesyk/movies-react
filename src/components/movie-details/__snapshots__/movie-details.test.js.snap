// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails renders 1`] = `
<div
  className="movie-details container"
>
  <img
    className="movie-image"
    src="http://poster_path"
  />
  <div
    className="movie-info"
  >
    <div
      className="title"
    >
      title
    </div>
    <div
      className="rating"
    >
      8
    </div>
    <div
      className="tagline"
    >
      tagline
    </div>
    <div
      className="year"
    >
      2007
    </div>
    <div
      className="duration"
    >
      122
      min
    </div>
    <div
      className="overview"
    >
      overview
    </div>
  </div>
</div>
`;
